ui/src/app/pages/login/account-creation/account-creation.spec.tsx

```diff
@@ -102,81 +102,6 @@ it('should handle username validity if name is valid', () => {
   expect(wrapper.exists('#usernameError')).toBeFalsy();
 });
 
-it('should handle invalid Email', () => {
```

Comment:
email is no longer handled in this file.  See account-creation-institution and its spec